src/Apps/__tests__/Fixtures/Artworks.ts

```diff
@@ -815,8 +815,11 @@ export const AllAdditionalDetailsPresent = {
   additional_information:
     "In Francesca DiMattio’s The Fox and the Hound, 2018, lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullam DiMattio’s The Fox and the Hound, 2018, lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullam. Tempor incididunt ut labore et dolore magnas.",
   sale: null,
+  isInAuction: false,
```

Comment:
What motivated these changes? Did you just notice that the stub was incomplete? Or is it related to some stories being changed in this diff?

Reply:
I rolled back these changes. Not sure if they're still needed.